tests/README.md

```diff
@@ -214,16 +198,58 @@ make test.kube.trafficmanagement
 
 1. If HUB and TAG aren't set, then their default values will match what is used by the prow tests.
 
-## Running Tests: go test
-
-You can execute individual tests using Go test as shown below.
-
-```bash
-make init
-export REPO_ROOT=$(git rev-parse --show-toplevel)
-go test ./tests/... -p 1  --istio.test.env kube \
-    --istio.test.ci --istio.test.work_dir <my_dir>
+## Adding A Setup Configuration
```

Comment:
This is advanced framework customization, not something we want to encourage test authors to do, so I don't think we want this in this README. How about moving it to another README in `tests/setup/`.

Reply:
Sounds good!